### PR TITLE
Fixing linters #195

### PR DESCRIPTION
**Pull request by glimchb**
- Fixing linters #195
- [storage] auto-generate protos

see https://linter.aip.dev/142/time-field-type
see https://linter.aip.dev/141/forbidden-types

### CODE REVIEW

File: storage/v1alpha1/frontend_nvme_pcie.proto [hunk 1]
```diff
@@ -125,7 +125,7 @@ message NVMeSubsystem {
     string model_number = 4;
 
     // maximum namespaces within a subsystem
-    int64 max_ns = 5;
+    int64 max_namespaces = 5;
```
Comment (glimchb) [inline storage/v1alpha1/frontend_nvme_pcie.proto:128]: see https://linter.aip.dev/142/time-field-type

File: storage/v1alpha1/common.proto [hunk 2]
```diff
@@ -18,15 +18,15 @@ option go_package = "github.com/opiproject/opi-api/storage/v1alpha1/gen/go";
 // virtual functions under the physical function.
 message NvmeControllerPciId {
     // Bus number, provided for future usage if needed. Currently set to ’0’
-    uint32 bus              = 1;
+    int32 bus              = 1;
```
Comment (glimchb) [inline storage/v1alpha1/common.proto:22]: see https://linter.aip.dev/141/forbidden-types

File: storage/v1alpha1/frontend_nvme_pcie.proto [hunk 3]
```diff
@@ -199,7 +199,7 @@ message NVMeNamespace {
 
     // 64bit Extended unique identifier for the namespace
     // mandatory if guid is not specified, optional otherwise
-    fixed64 eui64 = 8;
+    int64 eui64 = 8;
```
Comment (glimchb) [inline storage/v1alpha1/frontend_nvme_pcie.proto:202]: see https://linter.aip.dev/141/forbidden-types

**Review by seroyer — APPROVED** [review]
Progress!

**Pull request merged**